src/main/blackbox/blackbox_io.c

```diff
@@ -470,7 +470,7 @@ bool blackboxDeviceOpen(void)
      *
      * 9 / 1250 = 7200 / 1000000
      */
-    blackboxWriteChunkSize = MAX((masterConfig.looptime * 9) / 1250, 4);
+    blackboxWriteChunkSize = MAX((GUARD_LOOPTIME * 9) / 1250, 4);
```

Comment:
Ok you convinced me to do something with it. Looptime should be in there, but it will be rounded to the closest possible gyro refresh rate. Good idea actually!
On this way we can also easily experiment with different.

In the current version the lower (stable) looptimes really perform better without doubt. BTW not sure about the acc mode, but in rate mode it really gets better.

Can you be more specific about the ACC stuff. Especially this part " (maybe using filtered ACC data) and just integrate gyro reading on each loop"
BTW I am planning to leave the ACC part as it is in this PR as that requires a lot of personal research about that part of the code. I am sure it can be done more efficient.